rm_reporting/resources/response_chasing.py

```diff
@@ -72,7 +73,10 @@ def get(collection_exercise_id, survey_id):
                         "LEFT JOIN partysvc.respondent r ON e.respondent_id = r.id " \
                         "ORDER BY bd.sampleunitref;"
 
-        collex_details = engine.execute(text(collex_status))
+        try:
+          collex_details = engine.execute(text(collex_status))
+        except SQLAlchemyError:
+            logger.exception("SQL Alchemy query failed")
```

Comment:
Should we re-raise this exception or handle it in another way to indicate that the request failed?

Reply:
@benjefferies I've added re-raise after logging.